back/engines/commercial/report_builder/app/controllers/report_builder/web_api/v1/reports_controller.rb

```diff
@@ -61,14 +61,16 @@ def report
         end
 
         def create_params
-          @create_params ||= {
+          context_params = params.require(:report).permit(:participation_context_type, :participation_context_id)
```

Comment:
@luucvanderzee To create a report connected to a phase, you can add two new params: `participation_context_type` and `participation_context_id`. `name` is not required anymore (the doc will be updated soon) https://developers.citizenlab.co/api-docs/ee/frontweb_api/TAN-447-save-phase-reports/reports/create_a_phase_report.html